src/Ledger/Notation.lagda

```diff
@@ -0,0 +1,27 @@
+\section{Notation}
+\label{sec:notation}
+\begin{code}[hide]
+{-# OPTIONS --safe #-}
+
+module Ledger.Notation where
+\end{code}
+
+In this section, we introduce some notations used in this document.
+
+\begin{description}
+\item[Propositions, sets and types] This document loosly treats sets and types as the same thing.
+\item[Sums and products] The sum of \AgdaBound{A} and \AgdaBound{B} (also called their disjoint
+  union, coproduct, etc.) is denoted by \AgdaBound{A} \coproduct \AgdaBound{B}, and their product
+  is denoted by \AgdaBound{A} \agdatimes \AgdaBound{B}.
```

Comment:
I would rephrase this as "The product of  \AgdaBound{A} and \AgdaBound{B}  is denoted by \AgdaBound{A} \agdatimes \AgdaBound{B}; their sum (a.k.a. disjoint union or coproduct) is denoted \AgdaBound{A} \coproduct \AgdaBound{B}."

Reply:
I prefer the original order, since that's the order of the header as well. But I did slightly rephrase it.